gmeta/src/lib.rs

```diff
@@ -84,20 +210,24 @@ impl Types for () {
 }
 
 impl MetadataRepr {
+    /// Encode metadata representation into bytes.
```

Comment:
And similar flow for such docs
```suggestion
    /// Encode metadata into bytes.
```